tests/test_resource_upload.py

```diff
@@ -53,7 +55,7 @@
     {
         'mime_type': 'application/json',
         'sha1': 'e606bc6acc83666e1d40722e9c743a01e12e65ab',
-        'original_filename': 'example3.jpeg',
+        'original_filename': 'example4.jpeg',
```

Comment:
The duplicate that is included in the upload depends on the order that the resource files are read. - Here I just modified which file should be looked for manually based on which one was being uploaded.